packages/bits/spec/virtual-camera/PercyLens.ts

```diff
@@ -27,7 +27,7 @@ export class PercyLens implements ILens {
     public toolConfig() { console.warn("No config is availalbe for Percy"); }
```

Comment:
I just noticed that the filename doesn't follow the kebab-case naming convention